app/javascript/iiif.js

```diff
@@ -26,21 +21,25 @@ $(document).ready(function(){
         return canvas.startsWith('../') ? manifestUrl.replace('/manifest', canvas.slice(2)) : canvas;
       } else return null;
     }(new URL(document.location).searchParams);
-    const viewConfig = {};
+    const viewConfig = {
+      defaultView: 'single',
+      views: [
+        { key: 'single', behaviors: ['individuals'] },
+        { key: 'book', behaviors: ['paged'] },
+        { key: 'scroll', behaviors: ['continuous'] },
+        { key: 'gallery', behaviors: ['continuous', 'individuals', 'paged', 'unordered'] },
+      ],
+    };
     if (numChildren && numChildren === 1) {
       viewConfig.views = [
         { key: 'single' }
       ];
       viewConfig.defaultView = 'single';
     }
-    const culMiradorPlugins = flattenPluginConfigs(
-      canvasRelatedLinksPlugin, citationSidebar, hintingSidebar, miradorDownloadPlugins,
-      videoJSPlugin, viewerNavigation, viewXmlPlugin
-    );
     const foldersAttValue = miradorDiv.data('use-folders');
     const useFolders = (new Boolean(foldersAttValue).valueOf() && !String.toString(foldersAttValue).match(/false/i));
     if (useFolders) {
-      culMiradorPlugins.push(...collectionFoldersPlugin);
+      culMiradorPlugins.push([...Mirador.culPlugins.collectionFoldersPlugin]);
```

Comment:
It looks like you're adding an additional element to the culMiradorPlugins array and that element is an array.  Just to confirm, is that what's intended here?

Reply:
Just noting, because I've seen this a couple ways in the Mirador example code, that Mirador flattens out one level of plugin configs in this array. I think we probably want the simpler structure: `culPlugins` is an array, and then we push some elements onto the end of it here.   See also https://github.com/ProjectMirador/mirador/blob/main/src/extend/pluginPreprocessing.js#L16